src/main/java/com/icthh/xm/ms/entity/service/impl/XmEntityServiceImpl.java

```diff
@@ -528,7 +528,7 @@ public Page<XmEntity> searchV2(SearchDto searchDto, String privilegeKey) {
 
     @LogicExtensionPoint(value = "SearchByTemplate", resolver = TemplateTypeKeyResolver.class)
     @Override
-    @Transactional(readOnly = true)
+//    @Transactional(readOnly = true)
     @FindWithPermission("XMENTITY.SEARCH")
     @PrivilegeDescription("Privilege to search for the xmEntity corresponding to the query")
     public Page<XmEntity> search(String template,
```

Comment:
lep`s of this methods SearchByTemplate$$TEMPLATE_NAME.groovy can want to have transaction

Reply:
Why would they need a transaction? Do they update some entities during search operation?

Reply:
existing lep`s can read lazy relations inside this lep